cel/io.go

```diff
@@ -126,6 +128,55 @@ func ValueAsAlphaProto(res ref.Val) (*exprpb.Value, error) {
 	return alpha, err
 }
 
+// RPCStatusToEvalErrorStatus converts an RPC status to a celpb status. Status value is intended to
+// be wire and field compatible with `google.rpc.Status`.
+func RPCStatusToEvalErrorStatus(s *rpcpb.Status) (*celpb.Status, error) {
```

Comment:
Please remove this method. We don't need two levels of translation, just the one internal one.